lib/blockingproxy.ts

```diff
@@ -111,7 +111,6 @@ export class BlockingProxy {
   };
 
   handleProxyCommand(message, data, response) {
-    console.log('Got message: ' + message.url);
     var command = message.url.split('/')[2];
```

Comment:
Yeah, I never really took care of that when I ported to typescript. Let me clean all that up in a separate PR.